Development/nmos/settings.h

```diff
@@ -129,14 +129,19 @@ namespace nmos
         // this list is created and maintained by nmos::node_behaviour_thread; each entry is a uri like http://api.example.com/x-nmos/registration/{version}
         const web::json::field_as_value registration_services{ U("registration_services") };
 
-        // registration_heartbeat_interval [node]:
```

Comment:
See comments on **Development/nmos-cpp-node/config.json** and  **Development/nmos-cpp-registry/config.json**.